Package.swift

```diff
@@ -13,7 +13,13 @@ let package = Package(
     dependencies: [
         .package(url: "https://github.com/apple/swift-algorithms.git", from: "1.0.0"),
         .package(url: "https://github.com/adam-fowler/async-collections", from: "0.0.1"),
-        .package(url: "https://github.com/apple/swift-nio.git", from: "2.0.0"),
```

Comment:
This PR is in DRAFT and should NOT be merged until https://github.com/apple/swift-nio/pull/3271 is merged and released.

This needs updated with the latest swift nio version once 3271 is merged and in a release.